charts/config-server/templates/_overides.tpl

```diff
@@ -422,13 +422,13 @@ All env variables that are accessed from mosip config properties
 - name: SPRING_CLOUD_CONFIG_SERVER_OVERRIDES_ESIGNET_CAPTCHA_SITE_KEY
   valueFrom:
     secretKeyRef:
-      name: {{ .Values.overrides.secrets.captcha }}
+      name: {{ .Values.overrides.secrets.eCaptcha }}
       key: esignet-captcha-site-key
 
 - name: SPRING_CLOUD_CONFIG_SERVER_OVERRIDES_ESIGNET_CAPTCHA_SECRET_KEY
   valueFrom:
     secretKeyRef:
-      name: {{ .Values.overrides.secrets.captcha }}
+      name: {{ .Values.overrides.secrets.eCaptcha }}
       key: esignet-captcha-secret-key
```

Comment:
Please remove the esignet captcha block.
Add have to add it via esignet's install.sh